app/services/apollo.ts

```diff
@@ -80,6 +89,20 @@ export default class Apollo extends ApolloService {
       };
     });
   }
+
+  private createErrorLink() {
+    return onError(({graphQLErrors, networkError}) => {
+      if (graphQLErrors) {
+        graphQLErrors.forEach(({message, locations, path}) => {
+          Sentry.captureMessage(`${message}\n${locations}\n${path}`);
```

Comment:
Is `graphQLErrors` a list of "normal" errors (eg. an erroneous payload) or critical errors?

Reply:
From the docs:
> An array of errors from the GraphQL endpoint

It’s not really clear what ends up in there, I guess it’s not "critical" errors since the network layer didn’t fail…

Reply:
> In most cases, graphQLErrors is the errors field of the result from the last `next` call.

Ah, so this is related to the middlewares we add, if one of them returns an `error` field, this is what it will contain.

I think parsing errors and stuff like that can fall into that category. You think we should remove it?

Reply:
I don’t know… What kind of errors added by our middleware would we want to send to Sentry? If there are none, yes I think we should remove it 😄

Reply:
I read that when we use the httpLink, `graphQLErrors` is populated with `response.result.errors` so I guess we could remove it indeed 😄